### PR TITLE
CON-1337 add fonts to pattern library

### PR DESCRIPTION
**Pull request by Khatun**

### CODE REVIEW

File: assets/scss/_variables.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@ $SUSHI:            #8bc34a;
 $TROPICAL_BLUE:    #bfe8f6;
 $WHITE:            #fff;
 $BLACK:            #000;
+$ALABASTER:        #fafafa;
```
Comment (gideoncresswell) [inline assets/scss/_variables.scss]: The comments above this say no gray (grey) scale here. Should this go below and be `$gray-2:  darken($WHITE, 2%);     // #fafafa` instead?

File: assets/scss/_variables.scss [hunk 2]
```diff
@@ -52,7 +53,7 @@ $text-color-error:      $brand-danger;
 $link-color:            $brand-primary;
 $link-hover-color:      $brand-secondary;
 $disabled-gray:         $gray-40;
-
+$gray_vlight:           $ALABASTER;
```
Comment (gideoncresswell) [inline assets/scss/_variables.scss]: Change if using `gray-2` above

File: assets/scss/_fonts.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  font-family: "MuseoSans";
+  font-weight: 100;
+  src: url("/assets/fonts/museo/2D649C_4_0.eot");
+  src: url("/assets/fonts/museo/2D649C_4_0.eot?#iefix") format("embedded-opentype"), url("/assets/fonts/museo/2D649C_4_0.woff2") format("woff2"), url("/assets/fonts/museo/2D649C_4_0.woff") format("woff"), url("/assets/fonts/museo/2D649C_4_0.ttf") format("truetype"), url("/assets/fonts/museo/2D649C_4_0.svg#wf") format("svg");
+}
+
+@font-face {
+  font-family: "MuseoSans";
+  font-weight: 900;
+  font-style: Italic;
```
Comment (ChristopherGillis) [inline assets/scss/_fonts.scss]: Does this work with a capital "I"?

File: assets/scss/_variables.scss (hunk 1 above)
Comment (ChristopherGillis) [inline assets/scss/_variables.scss]: This is gray... use the grayscale vars...

**Pull request merged**